linera-chain/src/manager.rs

```diff
@@ -881,9 +842,9 @@ impl ChainManagerInfo {
             .map(|vote| Box::new(vote.value.clone()));
     }
 
-    /// Returns whether the `identity` is allowed to propose a block in `round`.
-    /// This is dependent on the type of round and whether `identity` is a validator or (super)owner.
-    pub fn can_propose(
+    /// Returns whether the `identity` is allowed to propose a block in `round`, except that it
+    /// returns `false` for super owners in single-leader rounds unless they are the leader.
```

Comment:
I find this comment quite confusing, I think it includes a double negation.

Alternative
```
/// Returns whether the `identity` is **allowed to propose a block** in `round`.
/// **Exception:** In single-leader rounds, a **super owner** is only allowed to propose if they are the designated **leader** for that round.
```